README.md

```diff
@@ -198,6 +198,7 @@ configuration:
 
 [git](http://git-scm.com/) configuration:
 
+- Adds a `co-upstream-pr $PR_NUMBER $LOCAL_BRANCH_NAME` alias to checkout remote upstream branch into a local branch.
```

Comment:
I believe this is a subcommand, not an alias.

```suggestion
- Adds a `co-upstream-pr $PR_NUMBER $LOCAL_BRANCH_NAME` subcommand to checkout remote upstream branch into a local branch.
```

Reply:
Changed it in be49b1df70e023cd80569cf53ffd4c9173d0e17b